pages/linux/terminator.md

```diff
@@ -0,0 +1,19 @@
+# terminator
+
+> Multiple GNOME terminals in one window.
+
+- Start with a fullscreen window:
+
+`terminator -f`
+
+- Split terminals Horizontally:
```

Comment:
Is there a reason to capitalize H and V ?

Reply:
It's because I just copy/paste it from `man` page :see_no_evil:, I will fix it.

Reply:
You're right, ok :ok_hand: